app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -34,6 +34,16 @@ class StudentForm extends React.Component {
       criminal_explanation: null,
       waiver_signature: null,
       waiver_date: null,
+      card_number: null,
+      cvc: null,
+      exp_month: null,
+      exp_year: null,
+      cardholder_name: null,
+      address_line1: null,
```

Comment:
can you add a stripe in front of these so it's not confused with the address of hte student so `stripe_address_line1` etc